libraries/Adapters/Microsoft.Bot.Builder.Adapters.Twilio/TwilioHelper.cs

```diff
@@ -30,19 +30,19 @@ internal static class TwilioHelper
         /// Creates Twilio SMS message options object from a Bot Framework <see cref="Activity"/>.
         /// </summary>
         /// <param name="activity">The activity.</param>
-        /// <param name="twilioNumber">The Twilio phone number assigned to the bot.</param>
+        /// <param name="twilioNumber">Optional. The Twilio phone number assigned to the bot. If not provided, defaults to Activity.From.Id to allow WhatsApp and other future integrations.</param>
         /// <returns>The Twilio message options object.</returns>
         /// <seealso cref="TwilioAdapter.SendActivitiesAsync(ITurnContext, Activity[], System.Threading.CancellationToken)"/>
-        public static TwilioMessageOptions ActivityToTwilio(Activity activity, string twilioNumber)
+        public static TwilioMessageOptions ActivityToTwilio(Activity activity, string twilioNumber = null)
```

Comment:
>string twilioNumber = null [](http://example.com/codeflow?start=79&length=26)

I see that you are not calling this from TwilioAdapter.SendActivities anymore, but is it used anywhere?

Reply:
Only in tests. It's not called anywhere else.